packages/perseus-editor/src/widgets/__tests__/interactive-graph-editor.test.tsx

```diff
@@ -130,12 +131,13 @@ describe("InteractiveGraphEditor", () => {
             },
         );
 
-        const defaultType = screen.queryByText("Linear function");
-        const otherType = screen.getByText("Polygon");
-
         // Assert
-        expect(defaultType).not.toBeInTheDocument();
-        expect(otherType).toBeInTheDocument();
+        await waitFor(() =>
```

Comment:
This will become a common theme throughout this PR. There are many places where our Perseus React components queue future renders through `setTimeout()` or `setState()` calls (or through a call to `reactRender()` helper). 

These renders do not happen unless we use `waitFor()` or the async `findBy...()` query methods. 

Without these, the tests either fail because the DOM has not been fully updated or testing-library spews warnings: `Warning: An update to Root inside a test was not wrapped in act(...).`